vaas/vaas/vcl/renderer.py

```diff
@@ -188,34 +191,58 @@ def __init__(self, varnish, input_data):
 
     def fetch_all_destinations_mappings(self, cluster: LogicalCluster, redirect: str,
                                         domain_mappings: list[DomainMapping]) -> tuple[str, list[str]]:
-        """Fetch tuple containing domain parsed from destination url and all found mappings for input cluster
+        """Fetch tuple containing domain parsed from destination url and all found mappings for input cluster.
+        Only non-regex mappings are considered here.
```

Comment:
NOTE: It means in the new "static_regex" mapping type, the domain is not replaced in "x-destination" header. So only relative destinations make sense in this type of mappings.

Or at least that's how I understand it. Let me be clear: I understand almost nothing here, so please double-check it :)